.github/pull_request_template.md

```diff
@@ -1,45 +1,19 @@
-<!--
-Thank you for submitting a PR to the rust rpm implementation!
--->
-
-# PR Content Desc
-
-<!---
-DELETE all that do not apply:
--->
-
-- 🩹 Bug Fix
-- 🦚 Feature
-- 📙 Documentation
-- 🦣 Legacy
-- 🪣 Misc
-
-<!---
-Mention the linked issue here.
-This will automatically close the issue once the PR is merged
-and creates a cross reference.
--->
-
-Closes #
```

Comment:
Ideally a user puts this in their commit message in which case it will show up here automatically, and this line will need to be deleted.

Reply:
Which unfortunately is rarely the case

Reply:
But how is asking them to do it there different from asking them to do it here?

Reply:
Because git doesn't ask them to :) this is an explicit request as part of the work flow to open a PR